src/ethereum/ethNetwork.js

```diff
@@ -127,6 +127,18 @@ async function broadcastWrapper(promise: Promise<Object>, server: string) {
   return out
 }
 
+type GetTxsParams = {
```

Comment:
Let's have a sidebar regarding this file

Reply:
As discuessed, we can update and further break up ethNetwork in a future PR